### PR TITLE
add class path resource resolver

### PR DESCRIPTION
**Pull request by MykolaGolubyev**

### CODE REVIEW

File: znai-docs/znai/flow/lookup-paths.md [hunk 1]
```diff
@@ -15,6 +15,17 @@ When you refer to a file using plugins like
 ../module/src/main/java
 ```
 
+# Class Path Lookup
+
+Znai is written using Java and can access resources in class path.
```
Comment (motlin) [inline znai-docs/znai/flow/lookup-paths.md:20]: access resources "from the classpath"

File: znai-docs/znai/flow/lookup-paths.md [hunk 2]
```diff
@@ -15,6 +15,17 @@ When you refer to a file using plugins like
...
+# Class Path Lookup
+
+Znai is written using Java and can access resources in class path. 
+Using class path lookup you can include snippets from projects deployed to, for example, Maven Central.
+
+:include-file: maven-class-path.xml {highlight: "classifier"}
+
+    :include-java: org/junit/Assert.java {entry: "assertArrayEquals"}
+
+:include-java: org/junit/Assert.java {entry: "assertArrayEquals"}
```
Comment (motlin) [inline znai-docs/znai/flow/lookup-paths.md:27]: How can I see how this renders?

File: znai-docs/znai/flow/lookup-paths.md (hunk 1 above)
Comment (MykolaGolubyev) [reply znai-docs/znai/flow/lookup-paths.md:20]: Could you please suggest the actual change so I can accept it?

File: znai-docs/znai/flow/lookup-paths.md (hunk 2 above)
Comment (MykolaGolubyev) [reply znai-docs/znai/flow/lookup-paths.md:27]: I don’t build docs during PRs yet. You will have to checkout and run build and then mvn:preview in the docs module

File: znai-docs/znai/flow/lookup-paths.md (hunk 1 above)
Comment (motlin) [reply znai-docs/znai/flow/lookup-paths.md:20]: Even though I'm a member, I don't think I have that permission.

File: znai-docs/znai/flow/lookup-paths.md (hunk 2 above)
Comment (MykolaGolubyev) [reply znai-docs/znai/flow/lookup-paths.md:27]: image

**Pull request merged**